Datagrid/SimplePager.php

```diff
@@ -13,6 +13,9 @@
 
 use Doctrine\Common\Collections\ArrayCollection;
 
+/**
+ * @deprecated Will be removed in version 2.0. Use Sonata\AdminBundle\Datagrid\SimplePager instead.
+ */
```

Comment:
Please trigger a deprecation error and add NEXT_MAJOR instructions. Also, mark corresponding tests as legacy. See CONTRIBUTING.md